jit_compile.cpp

```diff
@@ -22,12 +22,12 @@ void* jit_compile_and_load(
   char fdname[50];
```

Comment:
I think I over-engeneered it to avoid race a condition in using temp file. Maybe there is some better BKM to follow. We can remove fdname related code completely.